pastis/e2e_simulators/webbpsf_imaging.py

```diff
@@ -166,6 +163,34 @@ def set_up_nircam():
 
     return nircam, ote
 
+def set_up_cgi():
```

Comment:
Keep two blank lines before each new function definition - here, as well as further above in the script where they have been deleted.